data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Суон", "Свон" ],
+      [ "Лебланк", "Leblanc" ],
+      [ "Левеск", "Levesque" ],
+      [ "Лейн", "Lane" ],
+      [ "Ленд", "Land" ],
+      [ "Ландерс", "Landers" ],
+      [ "Лемонс", "Lemons" ],
+      [ "Фокс", "Fox" ],
+      [ "Литлджон", "Littlejohn" ],
+      [ "Лоуренс", "Lawrence" ],
+      [ "Медоуз", "Meadows" ],
+      [ "Луи", "Louis" ],
+      [ "Луу", "Luu" ],
+      [ "Боуман", "Bowman" ],
+      [ "Бест", "Best" ],
+      [ "Лэйси", "Lacey" ],
+      [ "Лэндри", "Landry" ],
+      [ "Лэрд", "Laird" ],
+      [ "Лэтэм", "Latham" ],
+      [ "Люнг", "Leung" ],
+      [ "Ма", "Ma" ],
+      [ "Майрик", "Myrick" ],
+      [ "МакАфи", "McAfee" ],
+      [ "МакГоверн", "McGovern" ],
+      [ "МакДональд", "McDonald" ],
+      [ "МакДонаф", "McDonough" ],
+      [ "МакДэниел", "McDaniel" ],
+      [ "МакИнтайр", "McIntyre" ],
+      [ "МакКарди", "McCurdy" ],
+      [ "МакКоллум", "McCollum" ],
+      [ "МакКрари", "McCrary" ],
+      [ "МакМахан", "McMahan" ],
+      [ "МакМахон", "McMahon" ],
+      [ "МакМиллан", "McMillan" ],
+      [ "МакНалти", "McNulty" ],
+      [ "МакНамара", "McNamara" ],
+      [ "МакНейр", "McNair" ],
+      [ "МакНелли", "McNally" ],
+      [ "МакНили", "McNeely" ],
+      [ "МакФадден", "McFadden" ],
+      [ "МакФарланд", "McFarland" ],
+      [ "МакФерсон", "McPherson" ],
+      [ "МакХью", "McHugh" ],
+      [ "Литтл", "Little" ],
+      [ "Маллинз", "Mullins" ],
+      [ "Маррокен", "Marroquin" ],
+      [ "Мартинс", "Martins" ],
+      [ "Мастерс", "Masters" ],
+      [ "Медейрос", "Medeiros" ],
+      [ "Медли", "Medley" ],
+      [ "Меза", "Meza" ],
+      [ "Мейерс", "Meyers" ],
+      [ "Мейо", "Mayo" ],
+      [ "Мейсон", "Mason" ],
+      [ "Мелансон", "Melanson" ],
+      [ "Мело", "Melo" ],
+      [ "Мелтон", "Melton" ],
+      [ "Мендес", "Mendes" ],
+      [ "Меррилл", "Merrill" ],
+      [ "Мерфи", "Murphy" ],
+      [ "Маркс", "Marks" ],
+      [ "Мизе", "Mize" ],
+      [ "Мирелес", "Mireles" ],
+      [ "Мозес", "Moses" ],
+      [ "Мойя", "Moya" ],
+      [ "Мок", "Mock" ],
+      [ "Янг", "Young" ],
+      [ "Хаммер", "Hammer" ],
+      [ "Монк", "Monk" ],
+      [ "Мониз", "Moniz" ],
+      [ "Монтанес", "Montanez" ],
+      [ "Монтеиро", "Monteiro" ],
+      [ "Морен", "Morin" ],
+      [ "Мориарти", "Moriarty" ],
+      [ "Фрост", "Frost" ],
+      [ "Морс", "Morse" ],
+      [ "Симен", "Seaman" ],
+      [ "Моултон", "Moulton" ],
+      [ "Мосс", "Moss" ],
+      [ "Мэйберри", "Mayberry" ],
+      [ "Мэйн", "Main" ],
+      [ "Мэйс", "Mayes" ],
+      [ "Мэллори", "Mallory" ],
+      [ "Мэсси", "Massey" ],
+      [ "Мэттсон", "Mattson" ],
+      [ "Наги", "Nagy" ],
+      [ "Наджера", "Najera" ],
+      [ "Надо", "Nadeau" ],
+      [ "Нг", "Ng" ],
+      [ "Невилл", "Neville" ],
+      [ "Викс", "Weeks" ],
+      [ "Нойманн", "Neumann" ],
+      [ "Барроуз", "Burrows" ],
+      [ "Ньюджент", "Nugent" ],
+      [ "Нэйлор", "Naylor" ],
+      [ "Нэпьер", "Napier" ],
```

Comment:
Apparently, this is transcribed primarily as [`Нейпир`](https://ru.wikipedia.org/wiki/%D0%9D%D0%B5%D0%B9%D0%BF%D0%B8%D1%80_(%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%B8%D1%8F)), of all things. There are alternatives, though far-fewer of them:

* [`Напьер`](https://ru.wikipedia.org/wiki/%D0%9D%D0%B0%D0%BF%D1%8C%D0%B5%D1%80)
* [`Напье`](https://ru.wikipedia.org/wiki/%D0%9D%D0%B0%D0%BF%D1%8C%D0%B5)
* [`Непер`](https://ru.wikipedia.org/wiki/%D0%9D%D0%B5%D0%BF%D0%B5%D1%80_(%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%B8%D1%8F))
* [`Нэпир`](https://ru.wikipedia.org/wiki/%D0%9D%D1%8D%D0%BF%D0%B8%D1%80,_%D0%9C%D0%B0%D0%BA%D0%B2%D1%8D%D0%B9)

The primary transcription is decidedly-English-sounding. The two after are mostly-French and French in sound, respectively. The other two... I don't even know; they sound nothing like the original, in any of its variations, in Russian.

If this were my effort alone, I would've gone with `Напьер` or something similar, since this is probably the closest to the original pronunciation.